workloads/generate_bolt/Dockerfile

```diff
@@ -154,7 +150,3 @@ RUN ninja bolt && ninja merge-fdata
 WORKDIR /home/${USERNAME}
 RUN sudo -E apt-get update && sudo -E apt-get upgrade -y && sudo -E apt-get install -y linux-tools-`uname -r`
 COPY --chown=${USERNAME}:root start.sh /home/${USERNAME}/start.sh
-```
-
-Please be noted the ghost workload needs DB to be prepared ahead of time. As in the user manual document in ghost folder, you need to fill in some data in the DB.
-You need to copy the `ghost.dump` and put the file into `genererate_bolt/common` folder.
```

Comment:
To make the dockerfile workable, please also prepare the ghost.dump file and put in the folder.